src/main.ts

```diff
@@ -1,4 +1,4 @@
-export { default, rebuildNativeModules } from './rebuild';
+export { rebuild, rebuildNativeModules } from './rebuild';
```

Comment:
@bendemboski Released a patch fix now to export on `default` and `rebuild` and manually define `__esmodule` so that babel picks up the default export